### PR TITLE
Run ghc for the wrapper in the temp directory to avoid .ghc-environment

### PR DESCRIPTION
**Pull request by hsenag**

### CODE REVIEW

File: src/HIE/Bios/Cradle.hs [hunk 1]
```diff
@@ -149,7 +149,9 @@ getCabalWrapperTool = do
         wrapper_hs <- writeSystemTempFile "wrapper.hs" cabalWrapperHs
         -- the initial contents will be overwritten immediately after by ghc
         wrapper_fp <- writeSystemTempFile "wrapper.exe" ""
-        callProcess "ghc" ["-o", wrapper_fp, wrapper_hs]
+        let ghc = (proc "ghc" ["-o", wrapper_fp, wrapper_hs])
+                    { cwd = Just (takeDirectory wrapper_hs) }
+        readCreateProcess ghc "" >>= putStr
```
Comment (mpickering) [inline src/HIE/Bios/Cradle.hs]: Is this `putStr` for debugging?

Comment (hsenag) [reply src/HIE/Bios/Cradle.hs]: It's to replicate the previous behaviour of `callProcess` which just passed through stdout directly. There doesn't seem to be a `callProcess`  equivalent if you need a custom `CreateProcess` value.

Comment (mpickering) [reply src/HIE/Bios/Cradle.hs]: You should use `createProcess` rather than `readCreateProcess` then? `proc` by default inherits the stdout handle.

Comment (hsenag) [reply src/HIE/Bios/Cradle.hs]: That would work, but then I'd have to copy and paste the bits of callProcess that throw an error if the command fails (or we decide we don't care about that).

**Pull request merged**